src/rt-app_parse_config.c

```diff
@@ -505,7 +505,7 @@ parse_thread_event_data(char *name, struct json_object *obj,
 		return;
 	}
 
-    if (!strncmp(name, "barrier", strlen("barrier"))) {
+	if (!strncmp(name, "barrier", strlen("barrier"))) {
```

Comment:
OK I'll split this and the other one into their own white space clean-up patch.

Reply:
This was split into its own commit too.

https://github.com/scheduler-tools/rt-app/pull/53/commits/76102ecddbafde72fa9a72c14025b2f270c6e16f

Can you clarify what 'remove' means here too please?

Reply:
We try to prevent mixing patches from different subject in the same patchset/pull in case some revert are needed as an example. These patches should be in a dedicated pull request

Reply:
I'm sorry that doesn't make sense to me as they are very tiny improvements. I'll drop them anyway. But I am afraid I won't be sending a new PR just for them.